tests/playground.html

```diff
@@ -19,6 +41,25 @@ <h1>Button Counter</h1>
             display.innerHTML = counter;
         });
     </script>
+
+    <script>
+    $(function() {
+        $("#visible").click(function() {
+            $('#invisible').toggleClass("show");
```

Comment:
Please add a place where we can extract the button states so our tests could be easily automated.

For example add a textarea with the states of all the buttons so after we click them we can get the text value of the textarea to validate the states easily.

Reply:
Understood.